packages/components/src/styles/Modal.module.css

```diff
@@ -0,0 +1,201 @@
+@import '../../../tokens/dist/media-queries.css';
+
+:root,
+[data-theme='default'] {
+  --lp-overlay-bg: #28282880;
+}
+
+[data-theme='dark'] {
+  --lp-overlay-bg: #282828b2;
+}
+
+@keyframes fade {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes zoom {
+  from {
+    transform: scale(0.8);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+@keyframes slide {
+  from {
+    transform: translateX(100%);
+  }
+
+  to {
+    transform: translateX(0);
+  }
+}
+
+@keyframes sheet {
+  from {
+    transform: translateY(100%);
+  }
+
+  to {
+    transform: translateY(0);
+  }
+}
+
+.base {
+  border-radius: var(--lp-border-radius-regular);
+  background-color: var(--lp-color-bg-overlay-primary);
+  outline: none;
+  padding: var(--lp-spacing-700);
+  box-shadow: 0 0 6px 0 #939598;
+
+  & [slot='title'] {
+    font: var(--lp-heading-2-medium);
+    flex-grow: 1;
+  }
+
+  & [role='dialog']:has(> [slot='body']) {
```

Comment:
This bit is a bit risky IMO: what if consumer wants/needs to add something around the body slot? Is the `>` necessary?

Reply:
Let me see if I can tweak that. Just need a check to make sure to only apply those styles if it has the slot present.